docs/translations/pt-BR/scripting/functions/AddCharModel.md

```diff
@@ -17,7 +17,7 @@ Adiciona um novo modelo de personagem personalizado para download. Os arquivos d
 | Nome    | Descrição                                                                                                                  |
 | ------- | -------------------------------------------------------------------------------------------------------------------------- |
 | baseid  | O ID do modelo da skin original a ser usado como base (personagens originais são usados caso o download falhar).           |
-| newid   | O ID do modelo da nova skin. Varia entre 20.000 a 30000 (10.000 slots) para serem usados posteriormente com SetPlayerSkin. |
+| newid   | O ID do modelo da nova skin. Varia entre 20.001 a 30000 (10000 slots) para serem usados posteriormente com SetPlayerSkin. |
```

Comment:
```suggestion
| newid   | O ID do modelo da nova skin. Varia entre 20001 a 30000 (10000 slots) para serem usados posteriormente com SetPlayerSkin. |
```

Reply:
`.` is correct for thousands separators in many languages.  The better change suggestion here would be switching `30000` to `30.000`.

Reply:
Ive added dots only on english (PT-BR dont use it on numbers >= 1000, only 2 decimals)